arrow-core.api

```diff
@@ -973,6 +967,7 @@ public final class arrow/core/extensions/ConstTraverseFilter$DefaultImpls {
 	public static fun foldLeft (Larrow/core/extensions/ConstTraverseFilter;Larrow/Kind;Ljava/lang/Object;Lkotlin/jvm/functions/Function2;)Ljava/lang/Object;
 	public static fun foldM (Larrow/core/extensions/ConstTraverseFilter;Larrow/Kind;Larrow/typeclasses/Monad;Ljava/lang/Object;Lkotlin/jvm/functions/Function2;)Larrow/Kind;
 	public static fun foldMap (Larrow/core/extensions/ConstTraverseFilter;Larrow/Kind;Larrow/typeclasses/Monoid;Lkotlin/jvm/functions/Function1;)Ljava/lang/Object;
+	public static fun foldMapA (Larrow/core/extensions/ConstTraverseFilter;Larrow/Kind;Larrow/typeclasses/Applicative;Larrow/typeclasses/Monoid;Lkotlin/jvm/functions/Function1;)Larrow/Kind;
```

Comment:
`foldMapA` seems to be added ???? Noticed this change too in `arrow-core-data`, is this a new method added in a typeclass?